custom-tests.json

```diff
@@ -591,6 +591,9 @@
     "SVGDescElement": {
       "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'desc'); if (instance.constructor.name === 'SVGElement') {return false;}"
     },
+    "SVGElement": {
+      "__base": "var instance = document.createElementNS('http://www.w3.org/2000/svg', 'unknown');"
```

Comment:
Yep, it's an `SVGElement`, says `Object.prototype.toString.call(document.createElementNS('http://www.w3.org/2000/svg', 'unknown'))`.